templates/default/event/eventlistfilter.html

```diff
@@ -1,4 +1,9 @@
 {block name=eventlistfilter}
+	<style>
+		.test span {
+			color: #888;
```

Comment:
Nie podoba mi się jakoś to  hardcodowanie koloru w samym szablonie HTML!
Ten kolor ma jakieś znaczenie? Może semantyczny styl przez klasę zrobić w style.less (jeśli jeszcze nie ma)?

Reply:
Nie ma. Przepraszam to zostało z moich testów i jest niepotrzebne. Zaraz puszczam poprawkę.